### PR TITLE
rewrite Zoila's protocol in MarkDown

### PR DESCRIPTION
**Pull request by murrayrm**
I've converted Zoila's protocol to MarkDown (instead of a Jupyter notebook that has MarkDown inside it).  I also added a few sections.  Mainly doing this as a demonstration/test of how to edit protocols in GitHub.

### CODE REVIEW

**Comment by murrayrm** [comment]
@zjuradoq Take a look at this version of the protocol and see if it looks OK.  Try doing a "code review" by making some comments on things.  I will update and then we can eventually merge the request.

File: txtl-liposome.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+7. Add 2 μL of reaction into 300 μL phospholipid solution, vortex for
+   30 s to create an emulsion
+
+8. Add the emulsion on top of the 20 μL centrifuge buffer, wait for the interface to stabilize and flatten between the emulsion and buffer
+
+9. Centrifuge at 11000 RPM for 30 s (tabletop centrifuge works well)
+
+10. Carefully remove the mineral oil, take 10 μL of the centrifuge
+    buffer (where the vesicles have been formed)
```
Comment (zjuradoq) [inline txtl-liposome.md]: Note: Use a new tip to take the centrifuge buffer

File: txtl-liposome.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+12. Mix gently, pipetting up and down with a new tip"
+
+## Step 3: Microscopy 
+13. Use Frameseal to make a small chamber on a coverslip
+
+14. Add 10 μL of the final solution to the inside chamber and seal
+    with cover slip
+
+15. Observe on microscope
+
```
Comment (zjuradoq) [inline txtl-liposome.md]: Image on 60X  RFP channel  (left) and bright field (right)

**Comment by zjuradoq** [comment]
I made some comments and added an image. I think with this method though as stated by Kate in #Slack that it may require 1 person as curator.

**Comment by murrayrm** [comment]
I saw the comments.  The image is just listed as a comment, but I will try to insert it into the protocol as a response.
Curation on github is easy: you can give one or more people admin access and they have the ability to merge changes.  So they are responsible for making sure proposed changes have been reviewed, meet community standards, etc (= curation!).

**Pull request merged**